src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -8447,7 +8447,7 @@ internal Arm64() { }
         ///   <para>void svst1_scatter[_u64base_f64](svbool_t pg, svuint64_t bases, svfloat64_t data)</para>
         ///   <para>  ST1D Zdata.D, Pg, [Zbases.D, #0]</para>
         /// </summary>
-        public static unsafe void Scatter(Vector<double> mask, Vector<ulong> addresses, Vector<double> data) => Scatter(mask, addresses, data);
```

Comment:
This (and others) includes a vector of addresses - `Vector<ulong> addresses`

Is it correct to remove the `unsafe` for these?